test/e2e/backup_script_test.go

```diff
@@ -0,0 +1,58 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"github.com/openshift/cluster-etcd-operator/test/e2e/framework"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+const (
+	masterNodeLabel = "node-role.kubernetes.io/master"
+)
+
+func TestBackupScript(t *testing.T) {
+	clientSet := framework.NewClientSet("")
+	masterNodes, err := clientSet.CoreV1Interface.Nodes().List(context.TODO(), metav1.ListOptions{LabelSelector: masterNodeLabel})
+	require.NoErrorf(t, err, "error while listing master nodes")
+
+	// create debug pod on first master node
+	debugNodeName := masterNodes.Items[0].Name
+	debugNodeAsArg := fmt.Sprintf("debug node/%s", debugNodeName)
+	err = exec.Command("oc", debugNodeAsArg).Run()
+	require.NoError(t, err)
```

Comment:
let's see if this passes, I believe this will tear down the pod as soon as the command finishes :)

Reply:
according to https://www.redhat.com/sysadmin/how-oc-debug-works, the pod is being created, think there is a timeout before it is being removed :/

Reply:
I think you need:
>     --preserve-pod=false:
>	If true, the pod will not be deleted after the debug command exits.

and instead of the chroot, you could also try:
>     --as-root=false:
>	If true, try to run the container as the root user

Reply:
done